tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/runtime/api/tests/Async.java

```diff
@@ -39,6 +39,26 @@
  */
 public class Async {
 
+    public static long getN(Environment env, BObject obj) {
```

Comment:
Shall we rename the function with a proper name? Maybe getClassVar, updateAndGetClassLevelVar